resources/lang/fre.ini

```diff
@@ -7,10 +7,78 @@ error = "Quelque chose s'est mal passé"
 noperm = "Vous n'avez pas la permission d'exécuter cette commande"
```

Comment:
Please make sure to remove entries that are not present in https://github.com/thebigsmileXD/MagicWE2/blob/translations/resources/lang/eng.ini